src/tersets.zig

```diff
@@ -27,6 +27,7 @@ pub const Error = error{
     IncorrectInput,
     NegativeErrorBound,
     OutOfMemory,
+    EmptySet,
```

Comment:
Maybe rename to `Empty` so it can be used by any custom collection implemented by the library.

Reply:
By using ArrayList for handling the upper and lower hull I dont need to define my own error anymore. I remove this.